### PR TITLE
Add libav-tools and LD_LIBRARY_PATHs needed for avprobe

### PR DESCRIPTION
**Pull request by zunda**
This change adds mp4 support, confirmed on Heroku-16 stack.

It seems that PaperClip makes use of `avprobe` command or `ffmpeg` command to deal with mp4 video files. While I could also confirm [adding ffmpeg](https://github.com/zunda/mastodon/tree/add-ffmpeg-on-heroku) works fine on Heroku, I bet going forward wit `avprobe` sounds better because 1) it requires less number of apt packages and 2) [it seems to use less memory (altough number of sample is pretty limited)](https://mastodon.zunda.ninja/@zundan/50339). I wonder what experts say.

### CODE REVIEW

File: .profile.d/mastodon-avprobe-library-paths.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/app/.apt/usr/lib/x86_64-linux-gnu/mesa:/app/.apt/lib/x86_64-linux-gnu:/app/.apt/usr/lib/x86_64-linux-gnu/pulseaudio
```
Comment (ykzts) [inline .profile.d/mastodon-avprobe-library-paths.sh]: memo: https://devcenter.heroku.com/articles/buildpack-api#profile-d-scripts

**Review by ykzts — APPROVED** [review]

**Comment by zunda** [comment]
Sorry for the noise again but PLEASE HOLD MERGING this pull request.

Reading [an article in Japanese](https://kledgeb.blogspot.jp/2015/11/ubuntu-1510-12-libavffmpeg.html), ffmpeg maybe better supported than libav. I'll come up with a pull request to use ffmpeg.

File: Aptfile [hunk 2]
```diff
@@ -1,2 +1,3 @@
 protobuf-compiler
 libprotobuf-dev
+libav-tools
```
Comment (ykzts) [inline Aptfile]: https://github.com/tootsuite/mastodon/pull/3274#issuecomment-303632986

**Review by ykzts — CHANGES_REQUESTED** [review]

**Comment by zunda** [comment]
Filed #3276

**Pull request closed without merging**